### PR TITLE
LPS-127758 using admin user to check if form was published

### PR DESCRIPTION
**Pull request by linolaoj**

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java [hunk 1]
```diff
@@ -509,10 +516,17 @@ public boolean isFormAvailable() throws PortalException {
 			if ((group != null) && group.isStagedRemotely()) {
 				ThemeDisplay themeDisplay = getThemeDisplay();
 
+				Role administratorRole = _roleLocalService.getRole(
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java:519]: @rijema or @linolaoj, please rename the variable to `role` to keep consistency with existing code. Brian probably will ask it for us. :wink:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java [hunk 2]
```diff
@@ -509,10 +516,17 @@ public boolean isFormAvailable() throws PortalException {
...
+					themeDisplay.getCompanyId(), RoleConstants.ADMINISTRATOR);
+
+				List<User> users = _userLocalService.getRoleUsers(
+					administratorRole.getRoleId());
+
+				User administratorUser = users.get(0);
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java:525]: @rijema or @linolaoj, please avoid create variables only to use as parameters of a method, instead use as parameter `users.get(0)`. According to Brian, usually, variables should be created when they need to be used on 2 or more places in the code. :wink:

**Review by natocesarrego — CHANGES_REQUESTED** [review]
Good job, guys! Please see my comments.

**Comment by rijema** [comment]
Here it is: https://github.com/natocesarrego/liferay-portal/pull/986 :)

**Pull request closed without merging**